learn/get-started/work-with-data-using-queries-in-ballerina.md

```diff
@@ -214,13 +344,60 @@ public function printErroneousData(table<CovidEntry> dataTable) {
     }
 }
 ```
-Here, you use the `sum` variable to hold the results of intermediate calculations in the query expression. 
+
+In this code:
+- If there is any record in which the number of reported `cases` is not equal to the sum of `recovered`, `active`, and `deaths`, this function will print it.
+- Here, you use the `sum` variable to hold the results of intermediate calculations in the query expression. 
 
 >**Note:** Even though this particular example uses a separate variable to demonstrate the usage of the `let` clause, you can do the calculations inline from within the `where` clause also.
 
-Now, let's call this function inside the `main` function. The final version of the `main` function looks like below.
+### Update the `main` function for finding
+
+To call the `findRecoveredPatientsOfCountries` function at the end to get the number of recovered patients, add the code below to the `main` function of the `main.bal` file. 
 
 ```ballerina
+printErroneousData(covidTable);
+```
+
+## The completed code for finding
+
+Below is the complete code for the finding function.
```

Comment:
```suggestion
Below is the complete code after adding the finding function.
```
We can also say like that and highlight the newly added logic. wdyt?

Reply:
Fixed.